File: exos/rust/scenario2/main.rs
use actix_web::{get, App, HttpResponse, HttpServer, Responder};

// Add an Endpoint Hello wich return Hello World ! 


#[actix_web::main]
async fn main() -> std::io::Result<()> {
    HttpServer::new(|| {
        App::new().service(hello)
    })
    .bind("127.0.0.1:8080")?
    .run()
    .await
}

File: exos/go/scenario2/main.go
// Comment créer une API web en Go avec le framework Gin
// https://www.thepolyglotdeveloper.com/2017/07/create-api-web-go-framework-gin/

package main

import (
	"github.com/gin-gonic/gin"
)

func main() {
	router := gin.Default()
	// Add an Endpoint Hello wich return Hello World ! 

	router.Run(":8080")
}
